### PR TITLE
User profile

### PR DESCRIPTION
**Pull request by GarrettCottrell**
--See profile data on dashboard for regular user
--Edit profile data
--Edit password
--Sad path for user attempts to update profile with duplicate email address or incorrect password

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -47,6 +47,14 @@
   get "/register", to: "users#new"
   post "/register", to: "users#create"
```
Comment (jakeheft) [inline config/routes.rb]: change post to '/users' instead of '/register' and corresponding route errors through app

**Review by jakeheft — APPROVED** [review]
Everything looks good after making the route change.

**Review by GregJMitchell — APPROVED** [review]
Looks great. Nice Restful routes!

**Comment by abcdefghijohn** [comment]
Love the RESTful routes guys!

**Pull request merged**